lib/uo_cli.rb

```diff
@@ -358,10 +363,32 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
         @scenario_folder = "#{@user_input[:scenario].split('.')[0].capitalize}"
         @scenario_path, @scenario_name = File.split(@user_input[:scenario])
         @feature_path, @feature_name = File.split(@user_input[:feature])
-        puts "\nAggregating results across all features of #{@feature_name} according to '#{@scenario_name}'...\n"
+        
         scenario_result = URBANopt::Scenario::ScenarioDefaultPostProcessor.new(run_func()).run
         scenario_result.save
-        puts "Done"
+        # FIXME: Remove this feature_reports block once urbanopt/urbanopt-scenario-gem#104 is merged
+        # save feature reports 
+        scenario_result.feature_reports.each do |feature_report|
+            feature_report.save_feature_report()
```

Comment:
you can remove this now !

Reply:
Leaving this block in because it doesn't work if I remove it.